src/share/config/ob_config_helper.cpp

```diff
@@ -476,6 +477,25 @@ bool ObConfigUseLargePagesChecker::check(const ObConfigItem& t) const
   return is_valid;
 }
 
+bool ObConfigClogDiskLimitSizeChecker::check(const ObConfigItem& t) const
```

Comment:
what is this function used for ?

Reply:
it is used to check if the value set by user is greater than disk size. If yes, it will return false and observer can't start normally.
Maybe I should print ERROR-log, just like the function 'ObConfigStaleTimeChecker::check'